app/javascript/mastodon/components/display_name.js

```diff
@@ -22,7 +22,7 @@ export default class DisplayName extends React.PureComponent {
 
     return (
       <span className='display-name'>
-        <bdi><strong className='display-name__html' dangerouslySetInnerHTML={displayNameHtml} /></bdi> {suffix}
+        <bdi><strong className='display-name__html' dangerouslySetInnerHTML={displayNameHtml}/></bdi> {suffix}
```

Comment:
```suggestion
        <bdi><strong className='display-name__html' dangerouslySetInnerHTML={displayNameHtml} /></bdi>
        <span>{suffix}</span>
```

Reply:
(need style)

Reply:
Thanks for your help :smile: 
Sorry I can't understand your comment above. Does `{suffix}` yet need style? Isn't it already inside `<span className='display-name__account'>` from line 20?

Reply:
Yes. It is necessary to add a style corresponding to that change.

**0001-Add-styles-to-suffix-of-display-name.patch**

```patch
From 557e5692a181ef8dae87ebd2020769cadd191faf Mon Sep 17 00:00:00 2001
From: Hayato <hayato.shimomae@dena.com>
Date: Mon, 22 Oct 2018 10:46:55 +0900
Subject: [PATCH 1/2] Add styles to suffix of display name

---
 app/javascript/mastodon/components/display_name.js | 2 +-
 app/javascript/styles/mastodon/components.scss     | 4 ++++
 app/javascript/styles/mastodon/rtl.scss            | 5 +++++
 3 files changed, 10 insertions(+), 1 deletion(-)

diff --git a/app/javascript/mastodon/components/display_name.js b/app/javascript/mastodon/components/display_name.js
index 31efab80a..f4448f442 100644
--- a/app/javascript/mastodon/components/display_name.js
+++ b/app/javascript/mastodon/components/display_name.js
@@ -23,7 +23,7 @@ export default class DisplayName extends React.PureComponent {
     return (
       <span className='display-name'>
         <bdi><strong className='display-name__html' dangerouslySetInnerHTML={displayNameHtml} /></bdi>
-        <span>{suffix}</span>
+        <span className='display-name__suffix'>{suffix}</span>
       </span>
     );
   }
diff --git a/app/javascript/styles/mastodon/components.scss b/app/javascript/styles/mastodon/components.scss
index f77dc405c..b677b5059 100644
--- a/app/javascript/styles/mastodon/components.scss
+++ b/app/javascript/styles/mastodon/components.scss
@@ -1538,6 +1538,10 @@ a.account__display-name {
   font-weight: 500;
 }
 
+.display-name__suffix {
+  margin-left: 5px;
+}
+
 .display-name__account {
   font-size: 14px;
 }
diff --git a/app/javascript/styles/mastodon/rtl.scss b/app/javascript/styles/mastodon/rtl.scss
index a2dfbf74b..76c755135 100644
--- a/app/javascript/styles/mastodon/rtl.scss
+++ b/app/javascript/styles/mastodon/rtl.scss
@@ -125,6 +125,11 @@ body.rtl {
     left: 0;
   }
 
+  .display-name__suffix {
+    margin-left: 0;
+    margin-right: 5px;
+  }
+
   .status__relative-time,
   .activity-stream .status.light .status__header .status__meta {
     float: left;
-- 
2.17.1 (Apple Git-112)

```

**0002-yarn-test-jest-updateSnapshot.patch**

```patch
From 1cca5e3c48e742c62d9faa568eabc87dcbe82595 Mon Sep 17 00:00:00 2001
From: Hayato <hayato.shimomae@dena.com>
Date: Mon, 22 Oct 2018 10:49:27 +0900
Subject: [PATCH 2/2] yarn test:jest --updateSnapshot

---
 .../__tests__/__snapshots__/display_name-test.js.snap | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/app/javascript/mastodon/components/__tests__/__snapshots__/display_name-test.js.snap b/app/javascript/mastodon/components/__tests__/__snapshots__/display_name-test.js.snap
index 29fdc2412..db47cb373 100644
--- a/app/javascript/mastodon/components/__tests__/__snapshots__/display_name-test.js.snap
+++ b/app/javascript/mastodon/components/__tests__/__snapshots__/display_name-test.js.snap
@@ -14,12 +14,15 @@ exports[`<DisplayName /> renders display name + account name 1`] = `
       }
     />
   </bdi>
-   
   <span
-    className="display-name__account"
+    className="display-name__suffix"
   >
-    @
-    bar@baz
+    <span
+      className="display-name__account"
+    >
+      @
+      bar@baz
+    </span>
   </span>
 </span>
 `;
-- 
2.17.1 (Apple Git-112)

```